src/commandline_options.cpp

```diff
@@ -510,6 +515,14 @@ commandline_options::commandline_options(const std::vector<std::string>& args)
 		windowed = true;
 	if(vm.count("with-replay"))
 		with_replay = true;
+	if(vm.count("wnoconsole"))
```

Comment:
I thought this was only supported on Windows?

Reply:
`--wconsole` got removed a while ago in the logging overhaul.

Reply:
How does that make sense? Right now, `--wconsole` is redundant if launched from the command line, because it already connects to the console if nothing is specified (if I'm understanding the code correctly). But what if you want to make a shortcut that launches Wesnoth with a console?

Reply:
It doesn't need an option for that. It attempts to attach to the parent process for its console and if that fails it creates its own.